apps/src/lib/ui/accounts/AddPasswordForm.jsx

```diff
@@ -19,12 +19,13 @@ const styles = {
   buttonContainer: {
     display: 'flex',
     alignItems: 'center',
+    justifyContent: 'flex-end',
```

Comment:
You might want to reuse the `.farSide` class that we use elsewhere on the Account page:

https://github.com/code-dot-org/code-dot-org/blob/6d4ea5b7ef8825a516384a9646ba0186dceb3271/dashboard/app/views/devise/registrations/edit.html.haml#L155-L157

...mostly because it accounts for RTL layout:

https://github.com/code-dot-org/code-dot-org/blob/ad0b7fa88b2967bef974c6bb8a6b27cbe9e3b1d1/dashboard/app/assets/stylesheets/application.scss#L2592-L2597

Reply:
i verified that the alignment and styles are correct for right-to-left languages, but ended up adjusting the `buttonContainer` styles rather than using the `.farSide` class, which didn't work quite right for this use case (it looked a little wonky for a flex container with multiple children)